base/float.jl

```diff
@@ -844,8 +844,8 @@ end
 """
     floatmin(T = Float64)
 
-Return the smallest positive normal number representable by the floating-point
-type `T`.
+Not the inverse of `floatmax`! Return the smallest positive normal number
```

Comment:
I'm not sure starting a docstring with what a function isn't (and an exclamation mark) is appropriate. If anything, I'd first make the positive statement and then the negative one (in any case without the question mark).

Reply:
The `isless` and `Base.isgreater` use the same format, so should we remove that as well? I'm only trying to follow the same format. At least its best stated at the beginning so users have this in mind while going through the function.

Reply:
I'd write `Return the smallest positive non-subnormal number`, to give people reading this pause. They might not know that `normal` in the context of floating point numbers is a technical term, which `non-subnormal` probably communicates better.

Reply:
Yes to starting with what it does to (clarifying later) and referring only to "sub-normal". 

I think it should also explain what "sub-normal" means, very briefly, or link to such an explanation.

Reply:
Won't these make the functions a little bit more complex and complicative. I think the word "opposite" should be enough for the distinction from `floatmax`.

A very good idea, would be to go with `not the opposite of floatmax`, and then provide an extended help section using `??` with what @mcabbott as suggested.

Reply:
I think it's nowhere near long enough to need extended help.

The subnormal bit should not be hidden in brackets, it's an important part of the definition.

And again, you seem stuck on a particular meaning of opposite. It not obvious at all what "is not the opposite of `floatmax`!" is trying to say. You mean it's not minus floatmax?